index.html

```diff
@@ -4,52 +4,56 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Akcióink</title>
+    <title>Főoldal</title>
 </head>
 
 <body>
     <ul>
         <li>
-            <a href="/">Home</a>
+            <a href="/">Főoldal</a>
         </li>
         <li>
-            <a href="/pages/products.html">Products</a>
+            <a href="/pages/products.html">Termékeink</a>
         </li>
         <li>
-            <a href="/pages/orders.html">Orders</a>
+            <a href="/pages/orders.html">Rendelés</a>
         </li>
         <li>
-            <a href="/about.html">About</a>
+            <a href="/about.html">Kapcsolat</a>
         </li>
     </ul>
-    <h1>Fantastic Burger</h1>
-    <ul>
+    <h1 style="color: brown;" >Happy Burger</h1>
+
+    <img src="/img/welcome.webp" width="768" height="432" alt="Hopsz, nem jeleníthető meg a kép...">
```

Comment:
`alt` is for screen reader users as well, so it would be better to give it a meaningful description about the image.

Reply:
OK, I will fix it in the next version.